### ISSUE TITLE
Need this to autostart

### ISSUE THREAD

**Issue by Pringlesman1461** [djlastnight/KeyboardSplitterXbox#233]
I am using this with a 4 player arcade.  I need this to enable itself when it's started and to auto start on boot up on windows.  Would this be possible?

**Comment by RudraPS97** [comment]
https://support.microsoft.com/en-us/windows/add-an-app-to-run-automatically-at-startup-in-windows-10-150da165-dcd9-7230-517b-cf3c295d89dd

**Comment by Pringlesman1461** [comment]
Yeah, I figured that part on my own.  THe problem is when Keyboard splitter starts it doesn't auto enable the profiles.
